### ISSUE TITLE
Change slider to 4 buttons 25%-50%-75%-100%

### ISSUE THREAD

**Issue by LunaPolkadex** [Polkadex-Substrate/Polkadex-Open-Beta#254]
**By raising this issue, I agree to the terms and conditions presented by Polkadex here:**[Terms and Conditions](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta-Feedback/blob/master/Polkadex_Open_Beta_Program_-_Terms__Conditions.pdf) 

**Description** 

I find current orderbook slider very hard to use and make make it stop where you want.
I think is more useful 4 preset buttons with 25% - 50% - 75% - 100% like large exchanges have

onchainaddress:{esnpHCUii7PvrGr5d3r2LLSseXzbKteTAsxLGHhhptmL9m4uJ}

**Comment by nlinhmus** [comment]
I totally agree with this, it's difficult to move the slider to 50% position.
